### PR TITLE
[258] Adds index view for drawless group and nav links

### LINKED ISSUE #258: Add Drawless Group Navigation

### PR DESCRIPTION
**Pull request by spaceraccoon**
Resolves #258
- Add index view based on group report
- Add link to nav under students
- Add additional variables in drawless group controller
- Minor edits for rebase

### CODE REVIEW

**Comment by esoterik** [comment]
@orenyk I think this is good to go but you should give it a look through

File: app/controllers/drawless_groups_controller.rb [hunk 1]
```diff
@@ -5,7 +5,18 @@ class DrawlessGroupsController < ApplicationController
...
+  def show
+    @compatible_suites = Suite.available.where(size: @group.size)
+    @compatible_suites_no_draw = @compatible_suites.where(draw_id: [])
+    @compatible_suites_in_draw = @compatible_suites - @compatible_suites_no_draw
+
+    render layout: 'application_with_sidebar'
+  end
+
+  def index
+    @groups_by_size = Group.where(draw_id: nil).group_by(&:size)
```
Comment (orenyk) [inline app/controllers/drawless_groups_controller.rb]: Order this by size

File: app/views/application/_nav_admin.html.erb [hunk 2]
```diff
@@ -6,7 +6,13 @@
       <li><%= link_to 'New Draw', new_draw_path %></li>
     </ul>
   </li>
-  <li><a href="#">Students</a></li>
+  <li>
+    <a href="#">Special Groups</a>
+    <ul class="menu vertical">
+      <li><%= link_to 'All Special Groups', groups_path %></li>
+      <li><%= link_to 'New Special Groups', new_group_path %></li>
```
Comment (orenyk) [inline app/views/application/_nav_admin.html.erb]: Create special group

File: app/views/groups/show.html.erb [hunk 3]
```diff
@@ -11,6 +11,7 @@
           <% end %>
           <% if policy(@group).reject_pending? %>
             <%= '(' unless policy(@group).accept_request? %><%= link_to 'reject', draw_group_reject_pending_path(@draw, @group, params: { 'user_id' => r.id }), method: :put %>)
+            (<%= link_to 'Accept', draw_group_accept_request_path(@draw, @group, params: { 'user_id' => r.id }), method: :put %>)
```
Comment (orenyk) [inline app/views/groups/show.html.erb]: get rid of this change

File: spec/features/groups/request_to_join_spec.rb [hunk 4]
```diff
@@ -29,7 +29,7 @@
       Membership.create(group: group, user: user, status: 'requested')
       log_in group.leader
       visit draw_group_path(group.draw, group)
-      click_on 'accept'
```
Comment (orenyk) [inline spec/features/groups/request_to_join_spec.rb]: get rid of this change

**Pull request merged**